src/vs/vscode.d.ts

```diff
@@ -2972,6 +2972,37 @@ declare namespace vscode {
 	}
 
 	/**
+	 * An individual terminal instance within the integrated terminal.
+	 */
+	export interface Terminal {
+
```

Comment:
set properties we use stuff like `name: string` and add a `@readonly` tag, like so: https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts#L2843 or https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts#L2751 (actually misses `@readonly`)